src/N3Store.js

```diff
@@ -87,8 +87,8 @@ export default class N3Store {
   // Finally, `graph` will be the graph of the created quads.
   // If `callback` is given, each result is passed through it
   // and iteration halts when it returns truthy for any quad.
-  // If instead `array` is given, each result is added to the array.
-  _findInIndex(index0, key0, key1, key2, name0, name1, name2, graph, callback, array) {
+  // If instead `yieldResult` is given, each result is yielded by the generator.
+  *_findInIndex(index0, key0, key1, key2, name0, name1, name2, graph, callback, yieldResult) {
```

Comment:
Seems like we could instead implement this with a wrapper around `_callback` then.

Reply:
In a4cf8e7610bf6e130c30c9542035d078cddcb4c6.